include/elf/riscv.h

```diff
@@ -87,6 +87,8 @@ START_RELOC_NUMBERS (elf_riscv_reloc_type)
   RELOC_NUMBER (R_RISCV_SET8, 54)
   RELOC_NUMBER (R_RISCV_SET16, 55)
   RELOC_NUMBER (R_RISCV_SET32, 56)
+  RELOC_NUMBER (R_RISCV_DELETE, 57)
```

Comment:
if "RELOC_NUMBER (R_RISCV_DELETE, 57)" is not defined, the number of R_RISCV_32_PCREL will be 57. When riscv_reloc_type_lookup searches BFD_RELOC_RISCV_32_PCREL, it return &howto_table[57]. But howo_table[57] is used by R_RISCV_DELET.

Otherwise, skip over 57 in RELOC_NUMBER. And define R_RISCV_32_PCREL  to 58 is ok, too.